config/locales/client.en.yml

```diff
@@ -3309,7 +3310,7 @@ en:
         hot: "There are no hot topics."
         filter: "There are no topics."
         education:
-          topic_tracking_preferences: "You can view and change your new topic tracking settings in <a href=\"%{basePath}/my/preferences/tracking\">your preferences</a>."
+          topic_tracking_preferences: 'You can view and change your new topic tracking settings in <a href="%{basePath}/my/preferences/tracking">your preferences</a>.'
```

Comment:
I think you can undo the change here since there’s no difference.

Reply:
Oh, I had my editor set to auto-format. Sorry I didn't notice this unrelated change!